Makefile

```diff
@@ -27,11 +27,13 @@ docker-push:
 	else \
 		docker login; \
 	fi
-	docker tag hiboxsystems/marge-bot:$$(cat version) hiboxsystems/marge-bot:$(VERSION)
+
+	docker tag hiboxsystems/marge-bot:$$(cat version) hiboxsystems/marge-bot:latest; \
+	docker push hiboxsystems/marge-bot:latest; \
+
 	if [ "$(VERSION)" = "$$(cat version)" ]; then \
-		docker tag hiboxsystems/marge-bot:$$(cat version) hiboxsystems/marge-bot:latest; \
+		docker tag hiboxsystems/marge-bot:$$(cat version) hiboxsystems/marge-bot:$(VERSION);
 		docker tag hiboxsystems/marge-bot:$$(cat version) hiboxsystems/marge-bot:stable; \
+		docker push hiboxsystems/marge-bot:$(VERSION); \
 		docker push hiboxsystems/marge-bot:stable; \
-		docker push hiboxsystems/marge-bot:latest; \
 	fi
-	docker push hiboxsystems/marge-bot:$(VERSION)
```

Comment:
Thanks, took me a while to understand this but I think it got it now. :+1: You moved some of the tagging & pushing to be done unconditionally, and some others to be done inside the `if [ "$(VERSION)" = "$$(cat version)" ]` block.

The way that it previously would unconditionally tag and push `hiboxsystems/marge-bot:$(VERSION)` is something you want to avoid, I take it? To only tag+push if the version equals the content of the `version` file?

Reply:
Yeah, there are still room for a lot of improvements but this is a start at least. Now it's easier to push a `latest` tag from the latest master at any time with `make docker-push`, and then to push a stable release with e.g. `VERSION=0.13.0 make docker-push`.